src/components/badge/badge.tsx

```diff
@@ -20,7 +22,7 @@ export class Badge {
      * six characters.
      */
     @Prop({ reflect: true })
-    public label: number | string;
+    public label?: number | string;
```

Comment:
didn't [you guys say that](https://github.com/Lundalogik/lime-crm-building-blocks/pull/24#discussion_r1104502622) `?` are unnecessary because Stencil already considers these optional. If we want them to be required, we should just add `!` instead? 😅

Reply:
Yes! But... TypeScript thinks otherwise from the component's point of view with `strictNullChecks: true`. 

In other words, adding `?` doesn't make any difference for the consumer. But it gives us better typing when using `this.label`.

Reply:
It also makes a light difference for the docs. If we have neither `!` nor `?`, the docs show this:

![image](https://user-images.githubusercontent.com/154725/224955556-be33b72b-1402-4b65-8f23-7ba5e36229a1.png)

Ideally, we should fix that in Kompendium, and only show `Optional: true/false`, but I guess it's not been the highest of priorities.

Reply:
https://github.com/jgroth/kompendium/issues/120